### PR TITLE
array-manipulation-3 | Cristofer Martinez

### PR DESCRIPTION
**Pull request by CristoferMar**
array-manipulation-3

### CODE REVIEW

File: array-manipulation-3/src/difference.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//             break
+//     push curernt first item to empty array
+//     push each remaining item in second array into empty array
+
+function difference(first, second) {
+  var difArray = [];
+  var trmSecond = second;
+
+  for (var i = 0; i < first.length; i++) {
+    var post = 1;
```
Comment (codingcodymiller) [inline array-manipulation-3/src/difference.js:20]: This would be better as a Boolean

File: array-manipulation-3/src/unique.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// check each item in OG array
+//     check each item in new array
+//         if current OG item is eual to an item on new array
+//             break
+//         else push current OG item to new array
+// return new array
+
+function unique(array) {
+  if (array.length === 0) { return []; }
+  var Unique = [array[0]];
```
Comment (codingcodymiller) [inline array-manipulation-3/src/unique.js:14]: You should avoid capital letters at the beginning of variables, that sort of casing is meant to be preserved for Constructor Functions

File: array-manipulation-3/src/union.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/* exported union */
+
+// sorry Cody, I did the psudo coding for this. But with the git issues that arose, it all got deleted
+// so I'm redoing this assignment but just gonna code it this time around. My appologies
+
+function union(first, second) {
+  var united = first;
+  for (var s = 0; s < second.length; s++) {
+    var push = 1;
```
Comment (codingcodymiller) [inline array-manipulation-3/src/union.js:9]: This would be better as a Boolean

File: array-manipulation-3/src/unique.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  for (var i = 1; i < array.length; i++) {
+    var post = 1;
```
Comment (codingcodymiller) [inline array-manipulation-3/src/unique.js:16]: This would be better as a Boolean

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start on this! Some notes for you:

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**